__std__/nix/plutus/library/plutus-project.nix

```diff
@@ -33,6 +33,11 @@ let
       name = "plutus";
     };
 
+    shell = {
```

Comment:
We use an explicit invocation of `shellFor` which already sets this, so I think this does nothing?